components/Page/ModalBuilderInputTitle.vue

```diff
@@ -4,89 +4,179 @@
     prevent-close
     :ui="{ base: 'md:max-w-[750px]' }"
   >
-    <UCard
-      :ui="{
-        body: {
-          padding: 'sm:py-4 sm:px-0',
-        },
-        header: {
-          padding: 'sm:py-4 sm:px-6',
-        },
-        footer: {
-          padding: 'sm:py-4 sm:px-6',
-        },
-      }"
-    >
-      <template #header>
-        <div class="flex items-center justify-between">
-          <h3 class="font-roboto text-xl font-medium leading-8 text-green-800">
-            Isi Judul Section
-          </h3>
-          <UButton
-            color="gray"
-            variant="ghost"
-            icon="i-heroicons-x-mark-20-solid"
-            class="-my-1"
-            @click="$emit('close')"
-          />
-        </div>
-      </template>
+    <UForm :schema="formSchema" :state="form" @submit="onSave">
+      <UCard
+        :ui="{
+          body: {
+            padding: 'sm:py-4 sm:px-0',
+          },
+          header: {
+            padding: 'sm:py-4 sm:px-6',
+          },
+          footer: {
+            padding: 'sm:py-4 sm:px-6',
+          },
+        }"
+      >
+        <template #header>
+          <div class="flex items-center justify-between">
+            <h3
+              class="font-roboto text-xl font-medium leading-8 text-green-800"
+            >
+              Isi Judul Section
+            </h3>
+            <UButton
+              color="gray"
+              variant="ghost"
+              icon="i-heroicons-x-mark-20-solid"
+              class="-my-1"
+              @click="onCancel"
+            />
+          </div>
+        </template>
 
-      <section class="grid grid-cols-1 gap-y-4">
-        <img
-          src="/images/illustration-title-input.svg"
-          width="548"
-          height="345"
-          class="justify-self-center"
-          alt="Ilustrasi gambar faq"
-        />
-        <div class="mx-6 gap-y-[10px] px-4 py-[14px]">
-          <UFormGroup label="Judul (Opsional)">
-            <template #default>
-              <UInput placeholder="Masukkan judul" color="gray" />
-            </template>
-            <template #help> Tersisa 250 Karakter </template>
-          </UFormGroup>
-          <UFormGroup label="Deskripsi (Opsional)">
-            <template #default>
+        <section class="grid grid-cols-1 gap-y-4">
+          <NuxtImg
+            src="/images/illustration-title-input.svg"
+            width="548"
+            height="345"
+            class="justify-self-center"
+            alt="Ilustrasi gambar faq"
+          />
```

Comment:
use <img> tag for static content @doohanas it didn't work on staging/prod

cc @yoslie 
```suggestion
          <img
            src="/images/illustration-title-input.svg"
            width="548"
            height="345"
            class="justify-self-center"
            alt="Ilustrasi gambar faq"
          />
```